Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM registry.access.redhat.com/ubi9/ubi-minimal
```

Comment:
Please consider adding a tag to the base image.
With a tag, you make sure that you'll get the same base image on every build (thus the same software versions).
This makes the build more predictable.

Reply:
I see the benefit  you mention, but it also makes us "stuck" in the past and should anything break we would probably not know about it. 
I prefer to know as soon as things break.
This practice is also used on other projects in kubevirt org such as [kubevirt/kubesecondarydns](https://github.com/kubevirt/kubesecondarydns), [kubevirt/bridge-marker](https://github.com/kubevirt/bridge-marker), [kubevirt/ssp-operator](https://github.com/kubevirt/ssp-operator) etc..

Reply:
This is good for going forward, but what happens when you need to fix an old version?
The old code with the fix will be built and copied on to a new base image than the one it previously used.

Reply:
good point, but as this application does not use unorthodox features from this image base, I think it's safe to assume it's rare enough to ignore. 
But for the off chance, I'll add the ability to choose the tag as an env var so that our future selves would have an easier life using it.

Reply:
Added [here](https://github.com/kiagnose/kubevirt-dpdk-checkup/compare/6fb2fa3018a782b5a5c376eb66b55743d9f440bb..ed3f3d9cdec38970c06d397cbd8c528b858e9c88)